selfdrive/sensord/sensors/lsm6ds3_accel.cc

```diff
@@ -28,11 +144,29 @@ int LSM6DS3_Accel::init() {
     source = cereal::SensorEventData::SensorSource::LSM6DS3TRC;
   }
 
+  ret = perform_self_test(LSM6DS3_ACCEL_POSITIVE_TEST);
```

Comment:
let’s gate the self test behind an environment variable for this release. we should always run the test and log the result, but only assert if the environment variable is set. make sure to set it in the unit tests